invenio_app_ils/ui/src/invenio_app_ils/common/api/documents/document.js

```diff
@@ -45,9 +45,11 @@ const queryBuilder = () => {
 
 const list = query => {
   return http.get(`${documentURL}?q=${query}`).then(response => {
+    let totalHits = response.data.hits.total ? response.data.hits.total : 0;
```

Comment:
`let` -> `const`

I think `ES` always returns the `hits.total` even if 0 results or did you see a case that it does not?
Same for the other occurrences of this line in this PR.

Reply:
I think it was because of tests, because example data didn't have the hits.total

Reply:
Es is returning the total number for sure, if example data are not well formed I think we should fix them. Assuming we have the total key there the `totalHits` is redundant I think.